docs/finder/finderspawntab.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Open folders in new tab or new window | Finder
+description: Set whether folders open in a new tab or a new window
+head:
+  - - meta
+    - property: 'og:title'
+      content: macOS defaults > Finder > Open folders in new tab or new window
+  - - meta
+    - property: 'og:description'
+      content: Set whether folders open in a new tab or a new window
+---
+
+# Open folders in new tab or new window
+
+Set whether folders shown in the Finder open in a new tab or a new window when using `⌘ cmd`+`double-click`, and which option is shown in the context menu.
+
+- **Tested on macOS**:
+  - Sonoma
+- **Parameter type**: bool
+
+## Set to `true` (default value)
+
+Open folders in a new tab
+
+```bash
+defaults write com.apple.finder "FinderSpawnTab" -bool "true" && killall Finder
+```
+
```

Comment:
Please feel free to add the screenshots you took:

```suggestion

<img
  src="./images/finderspawntab/true.png"
  alt="Example output with value set to true"
  width="953" height="571" style="height: auto"
/>

```

This helps a lot to understand!